src/devtools/client/webconsole/actions/messages.js

```diff
@@ -34,7 +34,11 @@ export function setupMessages(store) {
     store.dispatch(onLogpointLoading(logGroupId, point, time, location));
   LogpointHandlers.onResult = (logGroupId, point, time, location, pause, values) =>
     store.dispatch(onLogpointResult(logGroupId, point, time, location, pause, values));
-  LogpointHandlers.clearLogpoint = logGroupId => store.dispatch(messagesClearLogpoint(logGroupId));
+  LogpointHandlers.clearLogpoint = logGroupId => {
+    store.dispatch(removeLogGroupId(logGroupId));
+    store.dispatch(messagesClearLogpoint(logGroupId));
```

Comment:
minor point, but it's okay to also listen for this action in the reducer and do the same thing. you might not need: removeLogGroupId